lib/inc/cpp-pcp-client/connector/connector.hpp

```diff
@@ -154,8 +159,14 @@ class LIBCPP_PCP_CLIENT_EXPORT Connector {
     ///
     /// Throws a connection_not_init_error in case the connection has
     /// not been opened previously.
+    /// Also rethrows any exception that can be thrown by
+    /// startMonitoring.
```

Comment:
We can tweak the docstring. What I meant it to mean is that stopMonitoring can thrown all the same exception types that startMonitoring can throw (due to handling everything from startMonitorTask).